app/controllers/api/transformation_mappings_controller.rb

```diff
@@ -49,5 +63,9 @@ def fetch_mapping_resource(href)
       raise "Invalid source or destination type #{resource_href.subject}" unless collection_config.collection?(resource_href.subject)
       resource_search(resource_href.subject_id, resource_href.subject, collection_class(resource_href.subject.to_sym))
     end
+
+    def fetch_transformation_mapping(type, id)
+      resource_search(id, type, collection_class(type))
```

Comment:
I'd drop this method since it's a one-liner and just call resource_search.

Reply:
Ok, makes sense